storage/src/lib.rs

```diff
@@ -31,7 +31,11 @@
 //! ### Example
 //!
 //! ```rust
-//! use crate::storage::{Storage, iroh::IrohStorage};
+//! pub mod storage;
+//! pub mod iroh;
+//!
+//! use crate::storage::Storage;
+//! use crate::iroh::IrohStorage;
```

Comment:
Seems kind of weird, but these comments need to include these changes for `make docs` to succeed.